src/diraccfg/cfg.py

```diff
@@ -945,7 +947,10 @@ def loadFromBuffer(self, data):
           currentlyParsedString = ""
           currentComment = ""
         elif line[index] == "}":
-          currentLevel = levelList.pop()
+          try:
+            currentLevel = levelList.pop()
+          except IndexError:
+            raise SyntaxError("The cfg file seems to close more section than it opens (i.e. to many '}' vs '{'")
```

Comment:
`SyntaxError` is the wrong exception type for this as it's for invalid Python code, not generic parsing errors. This should be `ValueError` or a subclass thereof:
```suggestion
            raise ValueError("The cfg file seems to close more sections than it opens (i.e. to many '}' vs '{'")
```